mainflux/bootstrap.js

```diff
@@ -0,0 +1,244 @@
+const axios = require('axios');
+
+class  Bootstrap{
+    //Bootstraps API Client
+    /**
+     * @class Bootstrap
+     * Bootstrap is used to manage bootstrap configurations.
+     * It is used to create, update, view and remove bootstrap configurations.
+     * It is also used to bootstrap a thing.
+     * @param {string} bootstraps_url - The url of the bootstraps service.
+     * @param {string} content_type - The content type of the request.
+     * @param {string} bootstrapsEndpoint - The endpoint of the bootstraps service which is 
+     * configs.
+     * @returns {Bootstrap} - Returns a Bootstrap object.
+     * 
+     */
+    constructor(bootstraps_url){
+        this.bootstraps_url = bootstraps_url;
+        this.content_type = "application/json";
+        this.bootstrapsEndpoint = "configs";
+    }
+
+    Create(config, token){
+        //Create a bootstrap configuration
+        /**
+         * @method Create - Create a new bootstrap configuration. 
+         * Some of the key data needed include the external_key and external_id which must be
+         * specific to the thing provided with the thing_id. Mind that every configuration 
+         * must have a specific thing_id.
+         * @param {object} config - The configuration object.
+         * @param {string} token - The token to be used for authentication.
+         * @example
+         * const config = {
+         *      "external_id": "345",
+         *      "external_key": "012",
+         *      "thing_id": "3d49a42f-63fd-491b-9784-adf4b64ef347",
+         *      "name": "thing_name"
+         *   } 
+         */
+        const options = {
+            method: "post",
+            maxBodyLength: Infinity,
+            url: `${this.bootstraps_url}/things/${this.bootstrapsEndpoint}`,
+            headers: {
+                "Content-Type": this.content_type,
+                Authorization: `Bearer ${token}`,
+            },
+            data: JSON.stringify(config),
```

Comment:
axios converts js objects to json so no need for JSON.stringify() look at other simillar cases